app/assets/stylesheets/_overrides.scss

```diff
@@ -44,9 +44,9 @@ a {
 }
 
 .lux-tabs-item {
-  background: #f8f9fa;
+  background: #e4e4e4;
```

Comment:
For the places in this PR where we have hex values, I'd love to instead have named variables in https://github.com/pulibrary/pulfalight/blob/main/app/assets/stylesheets/variables/colors.scss.

Reply:
Addressed this.

Reply:
@hackartisan I don't think we need to change the site link color. I'm a bit confused how the tabs have been passing accessibility tests but the sticky bar doesn't if they're consistent. But regardless, @taylor-yamashita should feel free to experiment with bg colors and link text colors to make this look good and pass contrast tests. (We can also make the link text white.) I see these more as buttons (or at least look more like buttons and less like links) and would love to see dividers between them. Same with the tabs.

Reply:
For now, I will experiment with the colors. If we add dividers between the buttons/links/tabs, maybe that can go in a separate ticket?

Reply:
I ended up using `$bleu-de-france-darker: #23578b;` (a slightly darker blue pulled from the lux palettes) for the links.